app/src/main/java/ee/ria/DigiDoc/android/main/settings/signing/SettingsFragment.java

```diff
@@ -18,6 +19,9 @@
 
 import com.takisoft.preferencex.PreferenceFragmentCompat;
 
+import java.util.List;
+import java.util.Locale;
+
```

Comment:
It seems these new imports can be removed

Reply:
Done